backend/apps/dashboard/views.py

```
@@ -320,47 +320,19 @@ def _calculate_field(self, user, field):
                 activity_days.add(timezone.localdate(dt))
 
             # Apply streak freezes to calculate streak days
-            today = timezone.localdate(timezone.now())
-            join_date = timezone.localdate(user.date_joined)
             streak_days = 0
             current_day = today
```

Ruff (0.15.20)

[error] Undefined name `today`

(F821)